packages/webpack-cli/lib/bootstrap.js

```diff
@@ -23,6 +24,11 @@ const runCLI = async (cliArgs) => {
         // If the unknown arg starts with a '-', it will be considered an unknown flag rather than an entry
         let entry;
 
+        // enable/disable colors
+        if (typeof parsedArgs.opts.color !== 'undefined') {
```

Comment:
```suggestion
        if (parsedArgs.opts.hasOwnProperty("color")) {
```

Reply:
Isn't it same ?

Reply:
nope, you can have a value `color: undefined`

Reply:
```js
 test = {
   colors: undefined
}

console.log(typeof test.colors === "undefined")
//output: true

console.log(test.hasOwnProperty("colors"))
//output true
```

Reply:
Can you commit suggestion ? I'm off my system.